deployment/stacks/imports_included/uploads.ts

```diff
@@ -0,0 +1,36 @@
+import { Construct } from "constructs";
+import { aws_s3 as s3, RemovalPolicy, Tags, Aws } from "aws-cdk-lib";
+
+interface createUploadsComponentsProps {
+  scope: Construct;
+  stage: string;
+}
+
+export function createUploadsComponents(props: createUploadsComponentsProps) {
+  const { scope, stage } = props;
+
+  const attachmentsBucket = new s3.Bucket(scope, "AttachmentsBucket", {
+    bucketName: `uploads-${stage}-attachments-${Aws.ACCOUNT_ID}`,
```

Comment:
Is this named to replicate it's current name? If so great!

Reply:
yes.